server/test/unit/server/async/test_app.py

```diff
@@ -0,0 +1,18 @@
+"""
+This module contains tests for the pulp.server.async.app module.
+"""
+import mock
+import os
+import unittest
+
+from pulp.server.constants import PULP_DJANGO_SETTINGS_MODULE
+
+
+class TestCeleryApp(unittest.TestCase):
+    """
+    Assert that the DJANGO_SETTINGS_MODULE environment variable is set.
+    """
+    @mock.patch('pulp.server.initialization.initialize')
+    def test_django_env_variable(self, mock_initialize):
+        from pulp.server.async import app  # noqa
```

Comment:
This should be moved up to the module itself. Having it here doesn't really guard against the problem that other test code could have imported a line of code that set the environment variable. If it was already modified having the import statement here versus at the top won't solve that problem. As such, moving it to the top would be good.